README.md

```diff
@@ -1,2 +1,19 @@
 # Snapfu Patches
-This repository contains patch files utilized by Snapfu used to update Snap repositories.
\ No newline at end of file
+This repository contains patch files utilized by Snapfu to update Snap projects. Patch files are ordered by framework and version. Patch versions are automatically created when a new version of Snap is released - this is done within the Snap monorepo repository.
+
```

Comment:
Should we mentions some requirements here? Ie. only Unix environment support?

Reply:
I will add that in the `run` section - as I believe that it is currently the only Unix environment restriction.

Reply:
I will also make mention of that in this top level readme.